tests/test_examples.py

```diff
@@ -188,14 +187,33 @@ def test_1d_optimization(SOLOVEV):
     with pytest.warns(UserWarning):
         eq.optimize(objective, constraints, options=options)
 
-    np.testing.assert_allclose(eq.compute("R0/a")["R0/a"], 2.5)
+    np.testing.assert_allclose(eq.compute("R0/a")["R0/a"], 2.5, rtol=2e-4)
 
 
 @pytest.mark.regression
-@pytest.mark.solve
-def test_1d_optimization_old(SOLOVEV):
+def test_1d_optimization_bounds():
+    """Tests 1D optimization for target aspect ratio within bounds."""
+    eq = desc.examples.get("SOLOVEV")
+    objective = ObjectiveFunction(AspectRatio(target=(2.4, 2.6)))
+    constraints = (
+        ForceBalance(),
+        FixBoundaryR(),
+        FixBoundaryZ(modes=eq.surface.Z_basis.modes[0:-1, :]),
+        FixPressure(),
+        FixIota(),
+        FixPsi(),
+    )
+    options = {"perturb_options": {"order": 1}}
+    with pytest.warns(UserWarning):
+        eq.optimize(objective, constraints, options=options)
+
+    np.testing.assert_allclose(eq.compute("R0/a")["R0/a"], 2.6, rtol=2e-4)
```

Comment:
Is this the right test for this? Since technically anything between 2.4 and 2.6 would be acceptable?

Reply:
yea it is unclear to me how this tests it, I guess just that larger aspect ratio should give lower F balance?

Reply:
I changed the test to solve an equilibrium with a target force balance of `-1e3 N <= F <= +1e3 N`, and check that the final residuals are all nearly 0. 

This works, but I should note that it makes things more challenging for the optimizer. In fact, giving `target=0` yields better results than `target=(-1e3, 1e3)` in this example. I think this is because the optimizer is losing information about the cost landscape when we make most of the residuals 0, and it struggles to minimize the remaining nonzero terms.

Reply:
Was this ever fixed? Right now it looks like the only bound is on the aspect ratio but the assertion is that its a particular value?

Reply:
This function `test_1d_optimization_bounds` was replaced with `test_solve_bounds`. I'm not sure why it isn't updating in this PR GUI here, but when I pull the code locally it is correct.

Reply:
Ah ok I see. thanks.